setup.py

```diff
@@ -19,19 +96,35 @@ def package_data(pkg, root_list):
     return {pkg: data}
 
 
-# Main ##############################################################
+VERSION = get_version('image_explorer', '__init__.py')
+
+if sys.argv[-1] == 'tag':
+    print("Tagging the version on GitHub:")
+    os.system("git tag -a %s -m 'version %s'" % (VERSION, VERSION))
+    os.system("git push --tags")
+    sys.exit()
+
+README = open(os.path.join(os.path.dirname(__file__), 'README.md'), encoding="utf8").read()
 
 setup(
     name='xblock-image-explorer',
-    version='2.1.0',
+    version=VERSION,
     description='XBlock - Image Explorer',
-    packages=['image_explorer'],
-    install_requires=[
-        'XBlock>=1.2',
-        'parsel>=1.6.0,<=1.6.0',
```

Comment:
@Agrendalath, FYI: it looks like removing of this constraint breaks this XBlock for Quince. Consider this traceback that I'm getting when trying to add it to a unit:
```python
2024-01-29 07:31:31,744 WARNING 13 [xblock.plugin] [user 112] [ip 91.196.101.94] plugin.py:144 - Unable to load XBlock 'image-explorer'                                                                                                                                                     
Traceback (most recent call last):                                                                                                                                                                                                                                                          
  File "/openedx/venv/lib/python3.8/site-packages/xblock/plugin.py", line 141, in load_classes                                                                                                                                                                                              
    yield (class_.name, cls._load_class_entry_point(class_))                                                                                                                                                                                                                                
  File "/openedx/venv/lib/python3.8/site-packages/xblock/plugin.py", line 70, in _load_class_entry_point                                                                                                                                                                                    
    class_ = entry_point.load()                                                                                                                                                                                                                                                             
  File "/openedx/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2518, in load                                                                                                                                                                                            
    return self.resolve()                                                                                                                                                                                                                                                                   
  File "/openedx/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2524, in resolve                                                                                                                                                                                         
    module = __import__(self.module_name, fromlist=['__name__'], level=0)                                                                                                                                                                                                                   
  File "/openedx/venv/lib/python3.8/site-packages/image_explorer/__init__.py", line 23, in <module>                                                                                                                                                                                         
    from .image_explorer import ImageExplorerBlock                                                                                                                                                                                                                                          
  File "/openedx/venv/lib/python3.8/site-packages/image_explorer/image_explorer.py", line 37, in <module>                                                                                                                                                                                   
    from parsel import Selector                                                                                                                                                                                                                                                             
  File "/openedx/venv/lib/python3.8/site-packages/parsel/__init__.py", line 16, in <module>                                                                                                                                                                                                 
    from parsel.selector import Selector, SelectorList  # NOQA                                                                                                                                                                                                                              
  File "/openedx/venv/lib/python3.8/site-packages/parsel/selector.py", line 48, in <module>                                                                                                                                                                                                 
    lxml_version = Version(etree.__version__)                                                                                                                                                                                                                                               
AttributeError: module 'openedx.core.lib.safe_lxml.etree' has no attribute '__version__'                                                                                                                                                                                                    
```

`etree.__version__` has been added in [parsel==1.7.0](https://github.com/scrapy/parsel/commit/f9db6d9aea0a99c95338333fffd497c6794562d0).

Reply:
Thanks for catching this, @0x29a! This is now fixed in [v2.2.1](https://github.com/openedx/xblock-image-explorer/releases/tag/v2.2.1).

Reply:
Thanks for fixing this so quickly, @Agrendalath!